rust/src/model/message.rs

```diff
@@ -21,9 +21,27 @@ use std::collections::HashMap;
 
 use crate::error::{ClientError, ErrorKind};
 use crate::model::common::Endpoints;
+use crate::model::message::MessageType::{DELAY, FIFO, NORMAL, TRANSACTION};
 use crate::model::message_id::UNIQ_ID_GENERATOR;
 use crate::pb;
 
+pub enum MessageType {
+    NORMAL = 1,
+    FIFO = 2,
+    DELAY = 3,
+    TRANSACTION = 4,
+}
+
+fn transform_to_pb_message_type(message_type: &MessageType) -> i32 {
```

Comment:
IMO, it would be better to implement From trait for &MessageType than offering a free function:
```
impl From<&MessageType> for i32 {
}
```

Reply:
Thanks a lot! I've found a more concise way to achieve the conversion. Please review the change again.

Reply:
Yes, it makes sense.